src/librustc_target/spec/x86_64_apple_darwin.rs

```diff
@@ -19,6 +19,9 @@ pub fn target() -> TargetResult {
         target_env: String::new(),
         target_vendor: "apple".to_string(),
         linker_flavor: LinkerFlavor::Gcc,
-        options: base,
+        options: TargetOptions {
+            target_mcount: "\01mcount".to_string(),
```

Comment:
Do you mean this?

```suggestion
            target_mcount: "\u{1}mcount".to_string(),
```

Reply:
Thank you! We should apply your suggestion to other platforms e.g. `\01__gnu_mcount_nc`, right?

Reply:
@JohnTitor  Right

Reply:
Okay, I fixed them.